opl/opl3.c

```diff
@@ -1375,10 +1375,10 @@ void OPL3_GenerateStream(opl3_chip *chip, Bit16s *sndptr, Bit32u numsamples)
         OPL3_GenerateResampled(chip, sndptr);
 
         sample = *sndptr * 4;
-        if (sample > SHRT_MAX)
-          sample = SHRT_MAX;
-        else if (sample < SHRT_MIN)
-          sample = SHRT_MIN;
```

Comment:
You could have included `<limits.h>`?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/829/commits/8661b1fe40d273fb93cbf0a00653ec64924f0f3f